pkg/kapp/resources/identified_resources_list_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+package resources
```

Comment:
Is there any particular reason for this test to be in `resources` vs `resources_test`?